test/workbox-core/node/controllers/test-WorkboxCore.mjs

```diff
@@ -27,4 +30,69 @@ describe(`WorkboxCore`, function() {
       expect(core.assert).to.not.exist;
     });
   });
+
+  describe(`core.logLevel (getter)`, function() {
+    it(`should initialise to 'verbose' log level for dev build`, async function() {
+      process.env.NODE_ENV = 'dev';
+
+      const coreModule = await import('../../../../packages/workbox-core/index.mjs');
```

Comment:
Just confirming: this taking advantage of `@std/esm`'s support for `import()`s, right? And it gets switched on at some point during the initialization of the test harness?

Reply:
You are correct. The magic trick is: `mocha --require @std/esm <Glob> <Args>` which is done behind the scenes of `gulp test`.